src/components/organisms/TimingEventModal/TimingEventModal.tsx

```diff
@@ -18,84 +17,102 @@ import { useRelatedVenues } from "hooks/useRelatedVenues";
 import { Modal } from "components/molecules/Modal";
 
 import { ButtonNG } from "components/atoms/ButtonNG";
+import { Dropdown } from "components/atoms/Dropdown";
 
 import "./TimingEventModal.scss";
 
 export type TimingEventModalProps = {
   show: boolean;
   onHide: () => void;
-  venueId: string | undefined;
   event?: WorldEvent;
-  template?: VenueTemplate;
-  venue: WithId<AnyVenue>;
+  venueId?: string;
   setEditedEvent: Function | undefined;
```

Comment:
[code smell] usually handlers are like `onEventEdit` and usually the `Function` is discouraged in favor of `(args, here)=>result` kind of type


```suggestion
  setEditedEvent: Function | undefined;
```

Reply:
I completely agree this is old code and should be fixed.